plugin/inlay_hint.py

```diff
@@ -9,6 +10,30 @@
 import uuid
 
 
+class LspToggleInlayHintsCommand(LspWindowCommand):
+    capability = 'inlayHintProvider'
+
+    def run(self) -> None:
+        InlayHints.toggle(self.window)
+        status = "on" if InlayHints.are_enabled(self.window) else "off"
+        sublime.status_message("Inlay Hints are {}".format(status))
+        for session in self.sessions():
+            for sv in session.session_views_async():
+                sv.session_buffer.do_inlay_hints_async(sv.view)
+
+
+class InlayHints:
```

Comment:
Naming is hard but this is a very generic class name for what it does.

I'm thinking `ShowInlineHintsOverride`?